internal/http/middleware.go

```diff
@@ -68,3 +111,21 @@ func errorMiddleware() echo.MiddlewareFunc {
 		}
 	}
 }
+
+func getMetricName(pathURL string) string {
+	pathLower := strings.ToLower(pathURL)
+
+	m := map[*regexp.Regexp]string{
+		regexp.MustCompile(`^get /something$`):             "fetch_something",
```

Comment:
for a boilerplate, if we want to use these example as reference, it's better to be as comments instead of real code.